### PR TITLE
fix(dashboard): changed h3 elements to h2

### LINKED ISSUE #42268: 9.1.3.1a/2.2 Modal Dialog "Customize" - It is recommended to change the "h3" heading to "h2" heading.
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-be75bbae42-v2-n2

9.1.3.1a/2.2 Modal Dialog "Customize" - It is recommended to change the "h3" heading to "h2" heading.

### PR DESCRIPTION
**Pull request by emoral435**
## 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
image | image

## 🚧 Summary
Changed the h3 headings in the customize modal popup to be h2

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/dashboard/src/DashboardApp.vue [hunk 1]
```diff
@@ -65,7 +65,7 @@
 
 		<NcModal v-if="modal" size="large" @close="closeModal">
 			<div class="modal__content">
-				<h3>{{ t('dashboard', 'Edit widgets') }}</h3>
+				<h2 class="customize-modal-popup-header">{{ t('dashboard', 'Edit widgets') }}</h2>
```
Comment (Pytal) [inline apps/dashboard/src/DashboardApp.vue:68]:
```suggestion
				<h2>{{ t('dashboard', 'Edit widgets') }}</h2>
```
Unused 😉

**Review by Pytal — APPROVED** [review]
Other than class removal

File: apps/dashboard/src/DashboardApp.vue (hunk 1 above)
Comment (emoral435) [reply apps/dashboard/src/DashboardApp.vue:68]: woops! Forget to delete that, gotcha!

**Review by susnux — APPROVED** [review]

**Pull request merged**